lib/ddtrace/contrib/elasticsearch/core.rb

```diff
@@ -25,6 +25,8 @@ def initialize(*args)
 
         def perform_request(*args)
           pin = Datadog::Pin.get_from(self)
+          return super(*args) unless pin && pin.tracer
```

Comment:
Since you are not using `args`, you can just have `*` as argument instead of `*args`, and call `super` without any argument so that it is called with the current function arguments.